### PR TITLE
Disable auto subscription, log ingestion function parameterization, replaced forEach with for .. of

### PR DESCRIPTION
**Pull request by kamaz**
Replaced outstanding `forEach` with `for..of` to fix promise errors. 

Added to optional properties *logIngestionFunctionName* and *disableAutoSubscription* for details refer to https://github.com/iopipe/serverless-newrelic-lambda-layers/issues/27

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,6 +145,25 @@ custom:
     prepend: true
 ```
 
+#### `logIngestionFunctionName` (optional)
+
+Only required if your New Relic log ingestion function name different from `newrelic-log-ingestion`.
```
Comment (mrickard) [inline README.md]: Suggested edit: 
```
Only required if your New Relic log ingestion function name is different from `newrelic-log-ingestion`.
```

Comment (kamaz) [reply README.md]: Thanks 😄 good suggestion. Updated.

**Comment by kamaz** [comment]
@mrickard would you be able to merge the PR?

**Comment by mrickard** [comment]
@kamaz Merged--thank you! (Release coming next.)

**Pull request merged**